scss/chart.scss

```diff
@@ -70,6 +70,12 @@ body {
 }
 
 .basemap-satellite, .basemap-dark, .basemap-odyssey, .basemap-decimal {
+    .gradient-legend .block .text  {
+        color: #eeeeee;
+        text-shadow: -1px -1px 0 #000, 1px -1px 0 #000, -1px 1px 0 #000, 1px 1px 0 #000;
+        background: rgba(0,0,0,0);
```

Comment:
The intention would be more clear if you would set `background: none`. `background` is a shorthand for setting all background properties, including `background-color`. So another alternative would be to use `background-color: transparent`.